### ISSUE TITLE
Regression in 0.6.4 vs 0.6.3 in jmReport erroring out

### ISSUE THREAD

**Issue by DKroot** [jmeter-gradle-plugin/jmeter-gradle-plugin#36]
I have build in place that regressed when I upgraded from 0.6.3 to 0.6.4 with jmReport erroring out:

```
Can't transform result
java.io.FileNotFoundException: false (The system cannot find the file specified)
```

Here is my configuration:

```
jmeter {
    // Produce simple table HTML report in jmReport.
    enableReports = true // default: false

    // Produce extended HTML reports in jmReport.
    enableExtendedReports = false // default: true

    // Arbitrary JMeter properties passed to JMeter via -J.
    jmUserProperties = jmeterProperties

    // Suppress using timestamp in the result in order to stabilize filename.
    resultFilenameTimestamp = "none"

    // Fail build on any JMeter errors or assertion failures
    ignoreErrors = false
    ignoreFailures = false
}
```

**Comment by foragerr** [comment]
Yup, I broke that while adding the custom XSLT. I apologize.
Temporary work around is to use a custom XSLT, I will release a fix for this today.

BTW: Did you have a chance to make changes to the XSLT to show url?

**Comment by foragerr** [comment]
fixed in 0.6.5

**Comment by DKroot** [comment]
Thanks, that worked! I have not tried a custom XSLT yet, but I'll certainly do when I have a chance.
